anvil-python/anvil/provider/local/commands.py

```diff
@@ -140,7 +138,6 @@ def register_cli(
         cluster.add_command(join)
         cluster.add_command(list)
         cluster.add_command(remove)
-        cluster.add_command(resize_cmds.resize)
```

Comment:
What was resize doing here? I can't see any logic to replace it, so I assume it was just not needed at all.

Reply:
This was accidentally inherited from sunbeam. It is OpenStack specific and we need to remove it as it makes no sense to have it on Anvil